lib/oxigraph/src/storage/memory.rs

```diff
@@ -30,15 +30,15 @@ pub struct MemoryStorage {
 
 struct Content {
     quad_set: DashSet<Arc<QuadListNode>, BuildHasherDefault<FxHasher>>,
```

Comment:
As a general design question beyond the bug fix, I wonder if a concurrent slab would be a good data structure here so replace the pointers by indexes to avoid the reference counter memory and coherence traffic?

Reply:
Yes! It's definitely a good idea! Will add it to my TOOD. Thank you!